tests/pass-dep/libc/libc-epoll.rs

```diff
@@ -167,7 +161,9 @@ fn test_epoll_ctl_del() {
     assert_ne!(res, -1);
 
     // Test EPOLL_CTL_DEL.
-    assert!(check_epoll_wait::<0>(epfd, &[]));
+    let res = unsafe { libc::epoll_ctl(epfd, libc::EPOLL_CTL_DEL, fds[1], &mut ev) };
+    assert_ne!(res, -1);
```

Comment:
Is there a reason you're not doing `assert_eq!(res, 0)` here? `epoll_ctl` is documented to return 0 on success, so it seems odd to make the test succeed for other values like 2 or -2.

Reply:
Yup this makes sense.